### ISSUE TITLE
Multi-Image Maps with markers unseparated

### ISSUE THREAD

**Issue by Missionary2112** [javalent/obsidian-leaflet#308]
## Issue
When I created a multi-image map, such as example below

```leaflet
id: multi-images
image: 
  - URL image 1
  - URL image 2
defaultZoom: 7.2
zoomDelta: .4
width: 60%
height: 60%
```

And I put in some markers in layer 1 and some in layer 2. When I return back to the page, all the markers appear in layer 1 (including markers from layer 2).

## Version
- Leaflet: 5.0.3
- MacOS Monterey: 12.5

**Comment by Adomin8er** [comment]
Has anyone had any updates with this? Mine is facing the same issue. I can't save markers to a second layer on my maps. It will always place them on the original layer.

**Comment by slider0203** [comment]
I am seeing the same thing in Obsidian Leaflet version 5.2.2. In my case I created four layers and put markers on each layer.

It looked fine until I reloaded the note. Then the top layer displays all the markers and no other layer is getting any. It doesn't matter if I switch which is the top layer. The new top layer will always show all the markers.

It doesn't appear to be a data issue. The markers look like it's saving the correct layer in the data.js file.
